src/frontend/sharedComponents/ActionTab/KeyboardAccessory.tsx

```diff
@@ -29,7 +29,11 @@ export const KeyboardAccessory: FC<KeyboardAccessory> = ({onPress, items}) => {
       </View>
       <View style={styles.flexRow}>
         {items.map((item, idx) => (
-          <Pressable onPress={onPress} key={idx} style={styles.accessoryIcon}>
+          <Pressable
+            onPress={onPress}
+            key={idx}
+            style={styles.accessoryIcon}
+            testID={`${item.testID}-keyboard`}>
```

Comment:
why add the `-keyboard` here? doesn't seem like the `testID` is used anywhere else, so i would expect it to be used as is for this prop.

Reply:
ah i see. `item.testID` is used in other components. this component just happens to accept the same interface. a little confusing but that makes more sense